### PR TITLE
Components

### PR DESCRIPTION
**Pull request by matoushybl**

### CODE REVIEW

**Comment by MatyasKriz** [comment]
That flag whether rCLI should use currentFolder or create one is a lifesaver!

File: Sources/reactant/InitCommand.swift [hunk 1]
```diff
@@ -1,56 +1,93 @@
...
-            let task = Process()
-            task.launchPath = "/usr/bin/env"
-            task.arguments = args
-            if let workingDir = workingDir {
-                task.currentDirectoryPath = workingDir
+    func execute() throws {
+
+        let config: ProjectConfiguration
+
+        if(argumentedInit.value) {
```
Comment (MatyasKriz) [inline Sources/reactant/InitCommand.swift]: Why is this condition surrounded by parentheses?

File: Sources/reactant/InitCommand.swift [hunk 2]
```diff
@@ -67,76 +104,56 @@ class InitCommand: Command {
...
-        }
-        if false && readBool(title: "Enable UI Tests?") {
-            let uiTestDir = workingDir.appending(outputDir).appending(components: "UITests")
-            targets.append(
-                ResolvedTarget(
-                    name: "\(productName)UITests",
-                    sources: Sources(items: [], root: uiTestDir),
-                    type: .uiTest)
-            )
+        if false && readBool(title: "Enable Cuckoo - boilerplate free mocking framework?") {
```
Comment (MatyasKriz) [inline Sources/reactant/InitCommand.swift]: Doesn't this effectively comment out the code inside the `if`? Why is it done so?

**Review by MatyasKriz — COMMENTED** [review]
Overall though I think it's ready to merge.

**Review by TadeasKriz — APPROVED** [review]

**Pull request merged**